src/GatewayWrapper.jsx

```diff
@@ -1,12 +1,32 @@
-const { targetComponent, targetProps, logOut, recordToC } = props;
+let {
+  targetComponent,
+  targetProps,
+  logOut,
+  tosDomainName,
+  termsCid,
+  privacyCid,
+  recordToC
+} = props;
+
+tosDomainName = tosDomainName ?? "https://near.infura-ipfs.io";
+termsCid = termsCid ?? "QmaE4P6fn2VSTgAJ4gdEmAh7Ywiw9J3TT8aRjMaFLdoDPX";
+privacyCid = privacyCid ?? "QmRiu67GedetzBMwKMfJkrKnFNDSs8KSUDPKEgXKN2hVqU";
```

Comment:
Is it worth to store all this data here? I think it's better to store in .env file in near-discovery and pass it to this component like I did [here](https://github.com/near/near-discovery/issues/727) ?

Reply:
I agree, it probably makes most sense to have the gateway pass these values in. I left a comment on your other PR.